From c710ec3307aa069330489a4b005a662fbef77846 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Tue, 13 Aug 2019 14:49:22 +0200
Subject: [PATCH] Revert "turris1x-support: fix for kernel 4.19"

This reverts commit eb5f3e06b0ed3acd0db1511e3d47a7bdb3066947.

This patch is not compatible with kernel 4.14 so we revert it.
---
 hardware/turris1x/turris1x-support/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hardware/turris1x/turris1x-support/Makefile b/hardware/turris1x/turris1x-support/Makefile
index b5b78ab2e..3c8e8f42e 100644
--- a/hardware/turris1x/turris1x-support/Makefile
+++ b/hardware/turris1x/turris1x-support/Makefile
@@ -25,12 +25,12 @@ endef
 Build/Configure:=:
 
 define Build/Compile
-	$(KERNEL_MAKE) zImage turris-nand-bch.dtb
+	$(KERNEL_MAKE) zImage turris.dtb
 endef
 
 define Package/$(PKG_NAME)/install
 	$(INSTALL_DIR) $(1)/boot
-	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/turris-nand-bch.dtb "$(1)"/boot/fdt
+	$(INSTALL_DATA) $(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/turris.dtb "$(1)"/boot/fdt
 	mkimage -T script -C none -n boot -d files/boot.txt "$(1)"/boot.scr
 endef
 
-- 
2.22.0

